cli.js

```diff
@@ -1,27 +1,30 @@
 const yargs = require('yargs');
-const moniker = require('moniker');
 
 const logger = require('./lib/logger');
-const { makeNameValid } = require('./lib/nameUtil');
 const { deployHandler, createHandler, invokeHandler,
   logsHandler, loginHandler, removeHandler } = require('./lib');
 
+/**
+ * Prints the provided message(along with optionally displayed help)
+ * and then exits the process with a code of 1.
+ *
+ * @param message - string to print before exiting
+ * @param displayHelp - should help be displayed before your error?
+ */
+const msgAndExit = function msgAndExit(message, displayHelp) {
+  if (displayHelp) yargs.showHelp();
+  logger.error(message);
+  process.exit(1);
+};
+
 const handleCommand = async function handleCommand(options, specificHandler) {
-  const numArgs = options._.length;
-  if (numArgs > 1) {
-    yargs.showHelp();
-    logger.error('Too many positional args given for command');
-    process.exit(1);
+  // eslint-disable-next-line no-param-reassign
```

Comment:
Why is this needed, I don't see param reassignment here.

Reply:
🏓 ping?

Reply:
Sorry was busying focusing on other things, I'll remove it in a subsequent patch.